README.md

```diff
@@ -102,6 +102,7 @@ by the Go language:
 | [Shared Array Buffers              ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics)  |  N/A                                                        | Atomic Read-Modify-Write operations on a shared integer array              | De-facto standard supported by major JS implementations, C-like semantics                              | Target for Emscripten                                                        | N/A                                                          | N/A         |          2016 |           2017 | Yes    | V8, SM      |
 | [WebWorkers                        ](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API)                          |  Fork-Join                                                  | Messaging, shared array buffers                                            | De-facto standard supported in major browsers                                                          | Streaming media                                                              | N/A                                                          | Implicit    |          2010 |           2017 | Yes    | Chrome, FF  |
 | [Workerpool                        ](https://github.com/josdejong/workerpool)                                                   |  Fork-Join                                                  | N/A                                                                        | Small implementation                                                                                   | Asynchronous processing                                                      | Background worker                                            | N/A         |          2014 |           2017 | Yes    | V8          |
+| [Threads](https://github.com/devsnek/threads) | Fork-Join | N/A | Small Implementation | Asynchronous Processing | Distributed Calculations, threaded IO, sharding | No | 2018 | 2018 | Yes | V8 |
```

Comment:
Thanks for submitting `threads`, would you be willing to make the following changes?
- Insert it in alphabetical order
- `Worker Pool` is already claiming "Small implementation" as a "Unique Feature", is there anything that `threads` implements in a unique fashion, any particular capabilities that might cause someone to choose `threads` over `Worker Pool`?
- Insert spaces to match the column indenting used in the rest of the table
- Please update the `Intended uses`:
   = Threads are by definition not distributed, maybe `Parallel Calculations`?
   = Clarify how `Threaded IO` differs from Node's built-in asynchronous I/O
   = Sharding is a distributed database concept, how does `threads` implement this? 
- Use `N/A` instead of `No` for examples

Obviously I am very interested in parallel JS so I checked out `threads` but the documentation is sparse and didn't help me understand the design philosophy, likewise there weren't any examples to guide me.  Clearly a lot of work went into `threads` and I'd like to take advantage of it or at least better understand it.  Any additional documentation and/or examples you can add to `threads` will make it easier for new and curious users to get started.